client/services/FeedbackGeneratorService.js

```diff
@@ -280,6 +339,21 @@ tie.factory('FeedbackGeneratorService', [
     };
 
     /**
+     * Returns a string when it is detected that the user is unfamiliar
+     * with the programming language.
+     *
+     * @returns {String}
+     * @private
+     */
+    var _getUnfamiliarLanguageFeedback = function() {
+      return [
+        "Seems like you're having some trouble with this language. Why ",
+        "don't you take a look at the New to Python button at the bottom ",
```

Comment:
Should 'New to Python' be in quotes?

Should that string be pulled out into a constant and also used in the button text, so that we can ensure the two of them are always in sync?